src/app/Http/Controllers/ApiController.php

```diff
@@ -126,12 +129,79 @@ public function updateAvailability(Request $request, Location $location) {
     public function armorvax(Request $request) {
         $data = $request->all();
         $data_string = json_encode($data, JSON_PRETTY_PRINT);
-        $path = sprintf('armorvax/%s/u%d_%s.json', config('app.env'), $request->user()->id, date('Y-m-d_his'));
+        $data_collection = collect(json_decode($data_string));
 
-        // Write the full JSON data to a file on S3
-        \Storage::disk('s3')->write($path, $data_string);
```

Comment:
I think it would be good to keep this S3 write in here as a backup so that we don't lose anything ArmorVax posts even if there's an error during the update. You could wrap this in a try/catch to prevent this from causing the import to fail, and/or you could check if the s3 disk is valid with `if(config('filesystems.s3.key'))` to skip this for devs who don't have the key.

Reply:
Done! :-)